src/__tests__/createStore.test.js

```diff
@@ -75,4 +75,55 @@ describe('createStore', () => {
       expect(localStorage.getItem('browserstore_foo')).toBeNull()
     })
   })
+
+  const getMock = jest.fn((error, key) => {
+    if (key !== 'error') throw error
+  })
+  const setMock = jest.fn((error, key, data) => {
+    if (key !== 'error') throw error
+  })
+  const errorStoreAdapter = {
+    get(key) {
+      if (key === 'error') throw Error('Error getting key')
+    },
+    set(key, value) {
+      if (key === 'error') throw Error('Error setting key')
+    },
+    clear() {},
+    afterGet() {},
+    onGetError(error, key) {
+      getMock(error, key)
+    },
+    beforeSet(data) { return data },
+    onSetError(error, key, data) {
+      setMock(error, key, data)
+    },
+  }
+  const errorStore = createStore(errorStoreAdapter)
```

Comment:
That's a lot of boilerplate in the middle of the test cases, which makes the tests harder to read. I think it would work better if you put it at the top with the other stores.